src/ui/components/Playlists/Playlists.react.js

```diff
@@ -13,32 +16,42 @@ import AppActions from '../../actions/AppActions';
 |--------------------------------------------------------------------------
 */
 
-export default class Playlists extends Component {
+class Playlists extends Component {
   static propTypes = {
     params: PropTypes.object,
-    children: PropTypes.object,
     playlists: PropTypes.array,
     playerStatus: PropTypes.string,
   }
 
   constructor(props) {
     super(props);
+
+    this.autoRedirect = this.autoRedirect.bind(this);
+  }
+
+  componentDidMount() {
+    AppActions.library.setTracksCursor('playlist');
   }
 
   createPlaylist() {
     AppActions.playlists.create('New playlist', true);
   }
 
+  autoRedirect() {
+    return <Redirect to={`/playlists/${this.props.playlists[0]._id}`} />;
```

Comment:
[Redirect to object?](https://reacttraining.com/react-router/web/api/Redirect/to-object)

Reply:
I do not see the advantage here, Redirect to string seems more elegant to me